app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/activity/MainActivity.java

```diff
@@ -31,7 +31,20 @@ protected void onCreate(Bundle savedInstanceState) {
         BottomNavigationView navigation = findViewById(R.id.navigation);
         navigation.setOnNavigationItemSelectedListener(this);
 
-        loadFragment(new ExchangeFragment());
+        Intent intent = getIntent();
```

Comment:
What does this do? Can you add a comment in code for this function

Reply:
This is for when I want to load a specific tab and not default as it is exchange right now

Reply:
Ok, can you still add a comment as it is not explicitly clear